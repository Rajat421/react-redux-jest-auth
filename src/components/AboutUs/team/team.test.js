/**
 * Created by Counter on 7/2/2017.
 */
import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import TEAM from './team'

test('TEAM component should render as expected',()=>{
    const component =shallow(<TEAM/>);
    const tree =toJson(component);
    expect(tree).toMatchSnapshot()
});