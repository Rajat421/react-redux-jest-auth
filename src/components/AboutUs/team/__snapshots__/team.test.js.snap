// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEAM component should render as expected 1`] = `
<div>
  <section
    className="team section_padding"
  >
    <h3
      className="heading_h3"
    >
      Team
    </h3>
    <div
      className="team_container"
    >
      <div
        className="single_member"
      >
        <figure>
          <img
            alt=""
            src="member.jpg"
          />
        </figure>
        <h4
          className="membername"
        >
          Larry Page
        </h4>
        <h6
          className="designation"
        >
          Co-Founder of Google
        </h6>
      </div>
      <div
        className="single_member"
      >
        <figure>
          <img
            alt=""
            src="member.jpg"
          />
        </figure>
        <h4
          className="membername"
        >
          Larry Page
        </h4>
        <h6
          className="designation"
        >
          Co-Founder of Google
        </h6>
      </div>
      <div
        className="single_member"
      >
        <figure>
          <img
            alt=""
            src="member.jpg"
          />
        </figure>
        <h4
          className="membername"
        >
          Larry Page
        </h4>
        <h6
          className="designation"
        >
          Co-Founder of Google
        </h6>
      </div>
    </div>
  </section>
</div>
`;
